yarn-project/end-to-end/src/e2e_p2p/validators_sentinel.test.ts

```diff
@@ -15,6 +17,10 @@ const NUM_NODES = 4;
 const NUM_VALIDATORS = NUM_NODES + 1; // We create an extra validator, who will not have a running node
 const BOOT_NODE_UDP_PORT = 4500;
 const BLOCK_COUNT = 3;
+const EPOCH_DURATION = 5;
+const SLASHING_QUORUM = 6;
+const SLASHING_ROUND_SIZE = 10;
+const SLASH_AMOUNT = 1n;
```

Comment:
We should generally try much bigger values that are sane for our case to ensure that the things that are bigint are bigint and not numbers.